tests/testthat/test-verify.R

```diff
@@ -0,0 +1,41 @@
+testthat::test_that("verify returns the same object for empty teal_data", {
+  tdata1 <- teal_data()
+  testthat::expect_identical(tdata1, verify(tdata1))
+  testthat::expect_identical(tdata1@verified, TRUE)
```

Comment:
The latter is covered by the former: identity of all slots has already been tested.

Reply:
thanks, removed 019fc32e1fd424524237dd008dc2460a775c44f2